src/Symfony/Component/BrowserKit/Request.php

```diff
@@ -16,37 +16,31 @@
  */
 class Request
 {
-    protected string $uri;
-    protected string $method;
     protected array $parameters;
-    protected array $files;
-    protected array $cookies;
-    protected array $server;
-    protected ?string $content;
 
     /**
-     * @param string $uri        The request URI
-     * @param string $method     The HTTP method request
-     * @param array  $parameters The request parameters
-     * @param array  $files      An array of uploaded files
-     * @param array  $cookies    An array of cookies
-     * @param array  $server     An array of server parameters
-     * @param string $content    The raw body data
+     * @param string      $uri        The request URI
+     * @param string      $method     The HTTP method request
+     * @param array       $parameters The request parameters
+     * @param array       $files      An array of uploaded files
+     * @param array       $cookies    An array of cookies
+     * @param array       $server     An array of server parameters
+     * @param string|null $content    The raw body data
      */
-    public function __construct(string $uri, string $method, array $parameters = [], array $files = [], array $cookies = [], array $server = [], string $content = null)
+    public function __construct(
+        private string $uri,
+        private string $method,
+        array $parameters = [],
```

Comment:
I missed checking that on previous PRs on the topic, but here we can declare this argument using CPP.
Can you check of that's the case is other locations where we don't use CPP but we could, even if we assign the property in the body?
(The most common case where we cannot is when the argument accepts null but the property doesn't.)

Reply:
Ok for now I used it like "when we do anything with the given argument, don't Use CPP. Only Use CPP if we set the exact same value to the property"

so here you want to Use CPP for `$arguments`, even we are doing the `array_walk_recursive` right?

Reply:
Done ✅